rust/common-runtime/tests/js_environment.rs

```diff
@@ -0,0 +1,182 @@
+#![cfg(not(target_arch = "wasm32"))]
+
+mod shared;
+use anyhow::Result;
+use common_protos::{
+    common,
+    runtime::{self, runtime_client::RuntimeClient},
+};
+use common_tracing::common_tracing;
+use serde_json::json;
+use shared::start_runtime;
+
+#[tokio::test]
+#[common_tracing]
+async fn it_has_an_empty_import_meta() -> Result<()> {
+    const TEST_IMPORT_OBJECT: &str = r#"
+import { read, write } from "common:io/state@0.0.1";
+
+export const run = () => {
```

Comment:
Could move these to test fixtures. Could also have a single "environment" module with switch statement on input to probe the context to reduce module builds in tests as well

Reply:
Yeah, let's move it to `common-test-fixtures`! We already have another `common:module` fixture in there.